app/src/main/java/com/mashup/mobalmobal/fcm/MobalFirebaseMessagingService.kt

```diff
@@ -0,0 +1,45 @@
+package com.mashup.mobalmobal.fcm
+
+import android.app.NotificationManager
+import android.util.Log
+import androidx.core.content.ContextCompat
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+import com.mashup.mobalmobal.util.sendNotification
+
+
+class MobalFirebaseMessagingService : FirebaseMessagingService() {
+    private val TAG = "MyFirebaseMsgService"
```

Comment:
```suggestion
    companion object {
        private const val TAG = "MyFirebaseMsgService"
    }
```

이거 companion object로 빼주세요.